README.md

```diff
@@ -1,141 +1,60 @@
-# Australia API and Dashboard
+# Australian Bushfire API and Dashboard
 
 ## Background
 
-This project explores how wildfires have affected the country of Australia over time. Specifically, for this project, we gathered data captured by NASA satellites and various historical wildfire and climate data to help visualize what Australia not only has been through historically but also what Australia has gone through with the most recent (2019-2020) wildfire season. Go to our [website](https://australia-fire-api-dashboard.herokuapp.com/) to learn more about this project.
```

Comment:
I would like to keep this part:

Specifically, for this project, we gathered data captured by NASA satellites and various historical wildfire and climate data to help visualize what Australia not only has been through historically but also what Australia has gone through with the most recent (2019-2020) wildfire season.